redbot/core/core_commands.py

```diff
@@ -38,7 +38,7 @@
 
 log = logging.getLogger("red")
 
-OWNER_DISCLAIMER = (
+OWNER_DISCLAIMER = _(
```

Comment:
This needs a small bit of handling above it to prevent translating this during import.

Reply:
Woops. Fixed, I have deplaced it directly to the command.